### PR TITLE
zopfli@1.0.3.bcr.1

### PR DESCRIPTION
**Pull request by cj81499**
add zopflipng binary

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @alexeagle, modules you maintain (zopfli) have been updated in this PR. Please review the changes.

**Comment by cj81499** [comment]
```
BcrValidationResult.FAILED: zopfli@1.0.3.bcr.1 is using an unstable source url: `https://github.com/google/zopfli/archive/refs/tags/zopfli-1.0.3.tar.gz`.
You should use a release archive URL in the format of `https://github.com/<ORGANIZATION>/<REPO>/releases/download/<version>/<name>.tar.gz` to ensure the archive checksum stability.
See https://blog.bazel.build/2023/02/15/github-archive-checksum.html for more context.
```

https://buildkite.com/bazel/bcr-presubmit/builds/7724#0191f441-af30-4705-9c8e-662a5cf2298c/147-156

hmmm... I don't think zopfli publishes a "release archive". Would appreciate guidance on how to proceed. Thanks!

**Comment by cj81499** [comment]
Looks like there's a [`skip-url-stability-check` label](https://github.com/bazelbuild/bazel-central-registry/issues?q=label%3Askip-url-stability-check+) that needs to be added to this PR.

File: modules/zopfli/1.0.3.bcr.1/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
++    # avoiding https://github.com/bazelbuild/bazel-central-registry/issues/716
++    name = "zopfli",
++    srcs = glob(["src/zopfli/*.c"], exclude = ["**/*_bin.c"]),
++    hdrs = glob(["src/zopfli/*.h"]),
++    linkopts = ["-lm"],
++    strip_include_prefix = "src/",
++    visibility = ["//visibility:public"],
++)
++
++cc_binary(
```
Comment (fmeum) [inline modules/zopfli/1.0.3.bcr.1/patches/add_build_file.patch:15]: The targets below don't seem to be visible. Would users need them?

**Review by fmeum — COMMENTED** [review]
Cc @alexeagle

File: modules/zopfli/1.0.3.bcr.1/patches/add_build_file.patch (hunk 1 above)
Comment (cj81499) [reply modules/zopfli/1.0.3.bcr.1/patches/add_build_file.patch:15]: I _personally_ don't need them, but I can't speak for all use cases.

I _think_ they exist solely to service the `zopflipng` cli, but I admit I'm not totally certain of that.

**Review by alexeagle — APPROVED** [review]
thanks!

**Pull request merged**